packages/css/src/table.css

```diff
@@ -36,47 +35,23 @@
   }
 
   & > thead > tr:last-child > :is(th, td) {
-    border-bottom: var(--dsc-table-header-divider);
+    border-bottom: var(--dsc-table-border-divider);
   }
 
-  /* No border on last tbody row if followed by tfoot */
   & > tbody:has(+ tfoot) > tr:last-child > :is(th, td) {
-    border-bottom: none;
+    border-bottom: none; /* Skip border-bottom when followed by <tfoot> */
   }
```

Comment:
Footer should not have a border below, see figma:
![image](https://github.com/user-attachments/assets/d9ca192c-afba-4a48-989a-d928c20cd52a)

This only applies for the footer, last `tr` in `tbody` should have a border beneath

Reply:
It's weird that Chromatic did not mark this as changed, since the current implementation does not have a border at the bottom..

Reply:
Ah it did, but you accepted it. Please leave the UI review to reviewers next time 
https://www.chromatic.com/pullrequest?appId=66fe736b9d639fe6801bf130&number=3020&activeElementId=comparison-test-key-66fe7659c3c326f7dfdb9bf8-desktop&view=changes

Reply:
Lets check with @Febakke what is the intended design here.

Reply:
It depends on the context. I think this should be decided by the user of the table if they need a border under the footer or not.

Reply:
We decided on daily to do as it is in figma now:
- Border below last row in tbody
- No border below footer